python/GafferSceneUI/LightEditor.py

```diff
@@ -388,6 +388,12 @@ def __buttonPress( self, pathListing, event ) :
 					"command" : Gaffer.WeakMethod( self.__selectLinked )
 				}
 			)
+			menuDefinition.append(
```

Comment:
Might be worth having a divider here, to separate the innocuous "query the scene" items from the scary "delete stuff" one?

Reply:
Fixed in 66f23915ac7c889f901e7b526413234db8a794a1